minimal_dataset/README.md

```diff
@@ -0,0 +1,5 @@
+If using this data, please cite the original creators of the data from which this was subsetted ([Chandrasekaran et al., 2023](https://doi.org/10.1101/2023.03.23.534023)) and th$
```

Comment:
The other readme has great details! Would it make sense to distinguish this dataset with the other one somehow?